Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -4,14 +4,15 @@
 #include <chrono>
 #include <algorithm>
 #include <fstream>
+#include <iomanip>
 
 #include <thread>
 
 namespace Hazel {
 	struct ProfileResult
 	{
 		std::string Name;
-		long long Start, End;
+		long double Start, End;
```

Comment:
why long double ? Is this really archi independent ? Why not simply double ? https://en.cppreference.com/w/cpp/language/types say extended precision floating point type. Does not necessarily map to types mandated by IEEE-754. Usually 80-bit x87 floating point type on x86 and x86-64 architectures.

Reply:
Changed to use a custom chrono::duration with double for Start and changed End to chrono::microseconds ElapsedTime.  End was only used to determine elapsed time anyway.